docs/getting-started/dockstore-tools.rst

```diff
@@ -449,20 +455,25 @@ Recommended when you're not using Quay.io, someone else has the same
 tool name already and you want your own tool instead, or if you are not
 using build triggers.
 
-Converting Between Build Modes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Converting Between Build Modes (Quay.io Only)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Manual -> Fully-Automated:
 
-1. Deregister the manual tool
-2. Create a new Fully-Automated one (simply click "Refresh All")
+1. Add a git build trigger to the Quay.io repository
+2. Refresh the manual tool
 
 Fully-Automated -> Manual:
 
 1. Delete tool
-2. Create a new Manual tool (will have to recreate the Quay.io
+2. Create a new manual tool (will have to recreate the Quay.io
    repository)
 
+.. note::
+  When manually adding a Quay.io tool, if there exists a Fully-Automated tool
```

Comment:
It looks weird to have the Fully-Automated with leading capitals like that. I realize it was like that.